tests/letstest/scripts/test_leauto_upgrades.sh

```diff
@@ -105,9 +105,16 @@ if ./letsencrypt-auto -v --debug --version | grep "WARNING: couldn't find Python
     exit 1
 fi
 
-EXPECTED_VERSION=$(grep -m1 LE_AUTO_VERSION certbot-auto | cut -d\" -f2)
+# Once certbot-auto is deprecated on RHEL systems, we can unconditionally check
+# for INITIAL_VERSION below.
+if [ -f /etc/debian_version ]; then
+    EXPECTED_VERSION="$INITIAL_VERSION"
```

Comment:
Could you confirm we expect here, for any system running on Debian-based OSes, that `certbot-auto` will not do anything (and so keep the initial certbot version) because of the deprecation mechanism?

Here and below, I think it may worth a comment to explain this behavior (that will make also clearer what the RHEL deprecation will change), if the reader is not well aware of the `certbot-auto` deprecation mechanism.

Reply:
Sure I can add a comment here. Where were you thinking of adding one below?

Reply:
The "below" here was referring to the similar modification you did in `test_letsencrypt_auto_certonly_standalone`. I think that adding a line about the deprecation mechanism in the existing inline comment you wrote for both files about RHEL will do the trick.

Reply:
Got it. I added those comments.